cogs/send_introduction_reminders.py

```diff
@@ -134,6 +134,13 @@ async def send_introduction_reminders(self) -> None:
                     if message_contains_opt_in_out_button:
                         await message.edit(view=None)
 
+                if member not in guild.members:
+                    logger.info(
+                        "Member: %s left the server before the introduction "
+                        "reminder could be sent.",
+                    ), member
+                    repr(member)
+
```

Comment:
A `continue` statement is required so that the function does not continue with sending the message even when this check found that the message should not be sent